templates/display.html

```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
 </head>
 <body>
+    <a href="javascript:window.history.go(-1)">Back</a>
```

Comment:
```suggestion
    <a href="javascript:window.history.go(-1)">Back</a> <!-- Displayed on top of a potentially large JSON response, so it will remain visible -->
```

Reply:
Thanks for the feedback and yes I agree to this suggestion.